invenio_app_ils/ui/src/invenio_app_ils/common/components/ResultsTable/formatters.js

```diff
@@ -119,10 +119,15 @@ function formatSeriesToTableView(series) {
     Updated: toShortDate(fromISO(series.updated)),
   };
   if (!isEmpty(series.metadata)) {
-    serialized['Title'] = series.metadata.title;
-    serialized['Authors'] = series.metadata.authors;
+    serialized['Title'] = series.metadata.title.title;
```

Comment:
A bit nitpicky, but we could also check for title existence using has from lodash:
`if (has(series.metadata, "title.title"))`

Reply:
In general title is required in the schema so it should not go through validation if there is no title in title, and I would leave it for now because of the migration